src/main/java/thrift/weights.thrift

```diff
@@ -0,0 +1,28 @@
+/**
+ * The first thing to know about are types. The available types in Thrift are:
```

Comment:
Please remove these comments - I think you copied them from somewhere else? i32, double, list don't need documentation.